### ISSUE TITLE
dont want to use firebase

### ISSUE THREAD

**Issue by bendaacz** [bobangajicsm/react-portfolio-website#28]
Hi, I really dont want to use firebase in this project, it is possible to remove it? 

Thanks!

**Comment by AJAmit17** [comment]
Yes, you can add your own data and you can remove firebase as it's not that useful for a normal portfolio.

**Comment by bendaacz** [comment]
That is cool! Thank you.

Im new to ReactJS
